src/index.html.js

```diff
@@ -1,6 +1,6 @@
 import config from 'shared/config';
 
-export default function renderDocument({ helmet, rootHtml, buildManifest }) {
+export function renderInitialBlock({ helmet, buildManifest }) {
```

Comment:
I don't dislike the names. Here's a few more:

renderLeading and renderTrailing
openDocument, closeDocument

Reply:
I like the `open/closeDocument` direction. `block` is to generic
Or we could make it even more specific like `open/closeBody` and move the "root" div to the main render fn.

Reply:
Yes, `block` is too generic, i pushed this way to discuss this. I think without getting too verbose the best option is `open/closeDocument`.